classes/output/core/course_renderer.php

```diff
@@ -209,6 +218,10 @@ protected function coursecat_coursebox(coursecat_helper $chelper, $course, $addi
      * @return string
      */
     protected function coursecat_coursebox_content(coursecat_helper $chelper, $course) {
+        $theme = \theme_config::load('moove');
+        if(empty($theme->settings->courseboxview)){
```

Comment:
You have to follow the Moodle code standards. Here you need to add spaces after the **if** and before the **{**